crates/bevy_math/src/rotation2d.rs

```diff
@@ -0,0 +1,332 @@
+use glam::FloatExt;
+
+use crate::prelude::{Direction2d, Mat2, Vec2};
+
+/// A counterclockwise 2D rotation in radians.
```

Comment:
I'm assuming this means counterclockwise wrt an axis pointing into the screen, could it be clarified in the comment?

Reply:
It's not strictly constrained to "pointing into the screen", as I imagine this could be used for e.g. 2D physics in a 3D game (like a 2D minigame) or some other case where you embed 2D in 3D.

Essentially, this is just a counterclockwise rotation in a 2D plane in a right-handed Cartesian coordinate system, like this picture from Wikipedia:

![kuva](https://github.com/bevyengine/bevy/assets/57632562/527ef0b1-19d9-4c65-8dcb-810636e29930)

I'm not sure how I'd explain this clearly in a doc comment though without making it unnecessarily complicated for people. I feel like counterclockwise is pretty unambiguous in 2D, since a Z axis doesn't really even exist there. It's just the opposite of the direction in which the hands of a clock move when observing the clock from the front.

Reply:
> It's just the opposite of the direction in which the hands of a clock move when observing the clock from the front.

It's something like this I would have hoped to see in the docs, because then it's even more clear.

Reply:
This seems acceptable in the 2d case. In 3D we would need to note handedness.

Reply:
Yeah, but that is basically the definition of "counterclockwise", so I felt like it was kind of unnecessary. I can add something like that though.

Reply:
Counter-clockwise seemed unambiguous to me.

Reply:
I agree; I'll leave it as is for the time being since I'm not sure what I'd put there that would add value